### ISSUE TITLE
Adding disqus support?

### ISSUE THREAD

**Issue by mmoayyed** [apereo/apereo.github.io#21]
This is a discussion item, soliciting feedback from other authors and the universe at large on whether it makes sense for the blog to add/present support for https://disqus.com/. 

Thoughts and/or objections? 

Given this is more than a cosmetic change, I figured it deserved a few words before anything happens, though if it were it likely wouldn't take much:
https://github.com/apereo/apereo.github.io/blob/master/_config.yml#L36

**Comment by jimhelwig** [comment]
I am not familiar with disqus. Could you summarize a motivation for making this change?

**Comment by apetro** [comment]
:-1:. [@avoidcomments](https://twitter.com/avoidcomments) sums up my thoughts on the matter.

Don't read the comments. If anyone wants to comment, they can discuss on email lists, they can propose their own post, or post elsewhere, or comment on the pull requests whereby content is proposed into this site, or if the post is in some shared institutional voice propose edits to the post by means of Pull Request, or... .

Hypothesis: there's not a lot of untapped potential for value in comments on this site. To the extent that there's value in discussion, it's easily had on email list.

If any particular post garners such interesting comment that it's worth linking the relevant list threads, well, posts are easily edited to add links.

**Comment by mmoayyed** [comment]
Like @apetro's take on this. Less email lists though; more PRs or outside-email content is loads better I think. 

Jim, quick summary as Andrew also rather makes it helpfully obvious: allow readers to post comments on posts, is disqus.
